src/Controller/BillingController.php

```diff
@@ -39,6 +42,18 @@ class BillingController extends ControllerBase implements ContainerInjectionInte
    */
   public static $cachePrefix = 'apigee.monetization.billing';
 
+  /**
+   * The current user.
+   *
+   * @var \Drupal\Core\Session\AccountInterface
+   */
+  protected $currentUser;
+
+  /**
+   * @var \Drupal\Core\Form\FormBuilderInterface
```

Comment:
This. has to have a doc comment to pass PHPCS.

Reply:
Added.